src/main/java/BasicJavaHomeWork/Exercise_1.java

```diff
@@ -0,0 +1,17 @@
+package BasicJavaHomeWork;
+
+import java.util.Scanner;
+
+public class Exercise_1 {
```

Comment:
Rename please the class to be more representative of what is done here.

Reply:
done